src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -3072,6 +3072,7 @@ public enum PlatformID
         Unix = 4,
         Xbox = 5,
         MacOSX = 6,
+        Other = 7,
```

Comment:
Not related to this PR, but, @terrajobst, I see that the src has EditorBrowsableState.Never on some entries, but the ref doesn't have that.  Which is correct?

Reply:
It looks like they're supposed to be hidden in the ref:
https://github.com/dotnet/runtime/issues/21660
https://github.com/dotnet/corefx/pull/19697
It looks like they were removed as part of:
https://github.com/dotnet/runtime/pull/31991/files#diff-1d02aa10f6540e37197ff9f090ca1d9eR2944
https://github.com/dotnet/runtime/pull/31991/files#diff-5450da078ed8d6b7e0e7413e10ab68c6L953.
@jkotas, was that intentional?

Reply:
Fixing in https://github.com/dotnet/runtime/pull/39065

Reply:
It was not intentional. I guess I have just hit a GenAPI bug that I did not notice.